spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+RSpec.describe UsersController, type: :controller do
+  let(:params) { { id: other_user.id } }
+  let(:user) { FactoryBot.create(:user, :data_coordinator) }
+  let(:other_user) { FactoryBot.create(:user, organisation: user.organisation) }
+
+  before do
+    sign_in user
+  end
+
+  describe "GET #edit_password" do
+    it "returns not found" do
```

Comment:
This text could be a little more descriptive to make it clear that it's testing you can't edit the password of another user - it's not immediately obvious without reading into the test code itself